.env.default

```
@@ -19,3 +19,5 @@ NFT_FRAGMENTS_PER_QUERY=10
 
 SUBGRAPH_COMPONENT_QUERY_TIMEOUT=60000
 SUBGRAPH_COMPONENT_RETRIES=1
+
+THIRD_PARTY_PROVIDERS_RESOLVER_SERVICE=
```

Should we suffix the name with `URL` to make it clear what the expected value is?
```suggestion
THIRD_PARTY_PROVIDERS_RESOLVER_SERVICE_URL=
```